### PR TITLE
Changes to allow to load subresources on 403.

### PR DESCRIPTION
**Pull request by Natim**
First step in trying to address #124

### CODE REVIEW

File: scripts/reducers/bucket.js [hunk 1]
```diff
@@ -18,7 +18,10 @@ export function bucket(state = INITIAL_STATE, action) {
       return {...state, busy: action.busy};
     }
     case BUCKET_LOAD_SUCCESS: {
-      const {data} = action;
+      let {bid, data} = action;
+      if (typeof data === "undefined") {
```
Comment (n1k0) [inline scripts/reducers/bucket.js]: I think this should be processed in the reducer but rather in the `listBuckets` saga, when we receive the 403.

Comment (Natim) [inline scripts/reducers/bucket.js]: We should actually improve that patch because since we have a 403 we actually do not have it in listBuckets.

**Comment by Natim** [comment]
Actually I think the best course of actions would be to let user add buckets and before creating it we should try to list its collection.

So that if I want to open an existing bucket I do not have permission on but I have permission on its collection, I can use the create bucket or hidden bucket button to load it nevertheless.

**Comment by Natim** [comment]
For instance I cannot open https://form:c78110f1e7514ff5afd89ee28a568872@kinto.notmyidea.org/v1/buckets/formbuilder (403) but I can list https://form:c78110f1e7514ff5afd89ee28a568872@kinto.notmyidea.org/v1/buckets/formbuilder/collections

**Comment by n1k0** [comment]
Nopte that all of this would be solved if we had https://github.com/Kinto/kinto/issues/600, which may have just turned P1 with this issue ;)

File: scripts/reducers/bucket.js (hunk 1 above)
Comment (n1k0) [inline scripts/reducers/bucket.js]: Yeah I was more thinking of `loadRoute` actually.

**Comment by n1k0** [comment]
Fixed by #128.

**Comment by Natim** [comment]
Ok thanks.

**Pull request closed without merging**